client/my-sites/email/titan-new-mailbox/index.jsx

```diff
@@ -97,22 +93,12 @@ const TitanNewMailbox = ( {
 						</FormLabel>
 						{ hasNameError && <FormInputValidation text={ nameError } isError /> }
 					</FormFieldset>
-
-					{ showTrashButton && (
-						<Button
-							className="titan-new-mailbox__remove-mailbox-button"
-							onClick={ onMailboxRemove }
-						>
-							<Gridicon icon="trash" />
-							<span>{ translate( 'Remove mailbox' ) }</span>
-						</Button>
-					) }
 				</div>
 				<FormFieldset>
 					<FormLabel>
 						{ showLabels && translate( 'Email address' ) }
 						<FormTextInputWithAffixes
-							placeholder={ translate( 'Email' ) }
+							placeholder={ translate( 'Email address' ) }
```

Comment:
Just an idea but instead of repeating the label just above, we could actually provide the user with a suggestion: `e.g. contact`.  We already do that for [email forwards](https://github.com/Automattic/wp-calypso/blob/80f83f1f644748540ea07644f95dc583de38e19b/client/my-sites/email/email-forwarding/email-forwarding-add-new.jsx#L164-L166):

<img width="623" alt="screenshot" src="https://user-images.githubusercontent.com/594356/129581309-885be598-d0a3-4159-89e3-87d2e9a83e80.png">